src/toil/lib/conversions.py

```diff
@@ -1,8 +1,9 @@
 """Conversion utilities for mapping memory, disk, core declarations from strings to numbers and vice versa."""
-# TODO: Consolidate all conversion utilities here to use the same functions:
-#  src/toil/lib/humanize.py (bytes2human; human2bytes)
-#  src/toil/batchSystems/__init__.py (MemoryString)
-#
+from functools import total_ordering
+from typing import Optional, SupportsInt
+
+VALID_UNITS = ['b', 'k', 'm', 'g', 't', 'p', 'e', 'kb', 'mb', 'gb', 'tb', 'pb', 'eb',
```

Comment:
It might be worth making two lists so to reduce redundancy and use one in `bytes2human()`.  Something like:

```
BINARY_PREFIXES = []
BASETEN_PREFIXES = []
VALID_PREFIXES = BINARY_PREFIXES + BASETEN_PREFIXES
```